### DISCUSSION TITLE
翻译语言问题

### DISCUSSION THREAD

**Discussion by an2058185378** [pot-app/pot-desktop#704] [Q&A]
为什么翻译一篇文章有时候有那么一小段会变成英文，但是我的源语言是日文翻译目标第一语言是简中第二是繁中

**Comment by elarbor** [comment]
选择的是中文：
![image](https://github.com/user-attachments/assets/59825286-6027-478f-a7af-36c53cd6c2ee)


升级提示能不能也是中文啊？
![image](https://github.com/user-attachments/assets/4e589d15-3ed9-4d48-9386-df22438cbda5)

@Pylogmon
